src/main/java/com/blackducksoftware/integration/jira/config/controller/action/IssueCreationConfigActions.java

```diff
@@ -52,26 +59,30 @@
 
 public class IssueCreationConfigActions {
     private final BlackDuckJiraLogger logger = new BlackDuckJiraLogger(Logger.getLogger(this.getClass().getName()));
-    private final PluginSettingsWrapper pluginSettingsWrapper;
+    private final PluginConfigurationAccessor pluginConfigurationAccessor;
+    private final GlobalConfigurationAccessor globalConfigurationAccessor;
     private final AuthorizationChecker authorizationChecker;
     private final ProjectManager projectManager;
     private final WorkflowHelper workflowHelper;
     private final BlackDuckMonitor blackDuckMonitor;
     private final ProjectMappingConfigActions projectMappingConfigActions;
 
-    public IssueCreationConfigActions(final PluginSettingsFactory pluginSettingsFactory, final AuthorizationChecker authorizationChecker, final ProjectManager projectManager,
+    public IssueCreationConfigActions(final JiraSettingsAccessor jiraSettingsAccessor, final AuthorizationChecker authorizationChecker, final ProjectManager projectManager,
         final WorkflowHelper workflowHelper, final BlackDuckMonitor blackDuckMonitor) {
-        this.pluginSettingsWrapper = new PluginSettingsWrapper(pluginSettingsFactory.createGlobalSettings());
+        this.pluginConfigurationAccessor = jiraSettingsAccessor.createPluginConfigurationAccessor();
+        this.globalConfigurationAccessor = jiraSettingsAccessor.createGlobalConfigurationAccessor();
         this.authorizationChecker = authorizationChecker;
         this.projectManager = projectManager;
         this.workflowHelper = workflowHelper;
         this.blackDuckMonitor = blackDuckMonitor;
-        this.projectMappingConfigActions = new ProjectMappingConfigActions(pluginSettingsFactory, workflowHelper);
+        this.projectMappingConfigActions = new ProjectMappingConfigActions(jiraSettingsAccessor, workflowHelper);
     }
 
     public BlackDuckJiraConfigSerializable getCreator() {
         final BlackDuckJiraConfigSerializable txConfig = new BlackDuckJiraConfigSerializable();
-        final String creator = pluginSettingsWrapper.getIssueCreatorUser();
+
+        final PluginIssueCreationConfigModel issueCreationConfig = globalConfigurationAccessor.getIssueCreationConfig();
+        final String creator = issueCreationConfig.getGeneral().getDefaultIssueCreator();
```

Comment:
you could add some helper methods in GlobalConfigurationAccessor that calls these methods directly so you don't have to have as many dots.